WcaOnRails/spec/support/database_cleaner.rb

```diff
@@ -1,19 +1,15 @@
 RSpec.configure do |config|
-  reference_tables_to_keep = %w(Countries Continents Events Rounds Formats teams)
   config.before(:suite) do
-    DatabaseCleaner.clean_with(:truncation, except: reference_tables_to_keep)
-    reference_tables_to_keep.each do |table|
-      ActiveRecord::Base.connection.execute("TRUNCATE #{table};")
-      load "#{Rails.root}/db/seeds/#{table.underscore}.seeds.rb"
-    end
+    DatabaseCleaner.clean_with :truncation
+    TestDbManager.fill_tables
```

Comment:
Could we do `DatabaseCleaner.strategy = :truncation, { except: TestDbManager::CONSTANT_TABLES }` and then not fill the database since the data would have already been loaded by the seed database? Then we don't need the TestManager class at all?